### PR TITLE
Refactor code to modules

### PR DESCRIPTION
**Pull request by malukenho**

### CODE REVIEW

File: pkg/guard.go [hunk 1]
```diff
@@ -1,12 +1,12 @@
-package main
+package pkg
 
 import (
 	"errors"
 	"os"
 )
 
 //Checking that an environment variable is present or not.
```
Comment (batusa) [inline pkg/guard.go:8]: I think the function name and body content speaks for itself. 🔪

File: pkg/mocks.go [hunk 2]
```diff
@@ -1,7 +1,16 @@
-package main
+package pkg
 
 import "net/http"
 
+// HTTPClient interface
```
Comment (batusa) [inline pkg/mocks.go:5]: Same here?

File: pkg/mocks.go [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
```
Comment (batusa) [inline pkg/mocks.go:7]: We usually `make` or `send` a request.

Comment (malukenho) [reply pkg/mocks.go:7]: It is part of the original API 😬

https://golang.org/src/net/http/client.go?s=20406:20458#L574

Comment (batusa) [reply pkg/mocks.go:7]: Got it.

**Review by batusa — APPROVED** [review]

**Pull request merged**